plugins/orchestrator/README.md

```diff
@@ -70,9 +70,6 @@ backend:
     script-src: ["'self'", "'unsafe-inline'", "'unsafe-eval'"]
     script-src-elem: ["'self'", "'unsafe-inline'", "'unsafe-eval'"]
     connect-src: ["'self'", 'http:', 'https:', 'data:']
-orchestrator:
```

Comment:
We should keep `orchestrator:`. It's the parent of `sonataFlowService` and `dataIndexService`

Reply:
ack, on it...

Reply:
Addressed in 695641633fd287e14861c0c0a11b083783a7cd0c